libraries/WiFi/src/WiFiSTA.h

```diff
@@ -84,6 +84,7 @@ class WiFiSTAClass
     uint8_t subnetCIDR();
     
     bool enableIpV6();
+    bool IPv6(bool state);
     IPv6Address localIPv6();
```

Comment:
If possible, you should mark this as deprecated ... there should be no need for the client to have to handle the difference between IPv6 and not IPv6.

Reply:
Like above I would agree in theory, but existing code would brake. IPv6 should need to be explicitly accepted by the code running, or many things will break.